src/mann_whitney.jl

```diff
@@ -52,7 +71,17 @@ end
 
 
 ## EXACT MANN-WHITNEY U TEST
+"""
```

Comment:
Better attach this docstring to the constructor than to the type since it documents the function. Same below.